scaleway/resource_lb_frontend_beta.go

```diff
@@ -70,22 +152,27 @@ func resourceScalewayLbFrontendBetaCreate(d *schema.ResourceData, m interface{})
 		return err
 	}
 
-	var createReq = &lb.CreateFrontendRequest{
+	//loop through new acl
+	res, err := lbAPI.CreateFrontend(&lb.CreateFrontendRequest{
 		Region:        region,
 		LbID:          LbID,
 		Name:          expandOrGenerateString(d.Get("name"), "lb-frt"),
 		InboundPort:   int32(d.Get("inbound_port").(int)),
 		BackendID:     expandID(d.Get("backend_id")),
 		TimeoutClient: expandDuration(d.Get("timeout_client")),
 		CertificateID: expandStringPtr(expandID(d.Get("certificate_id"))),
-	}
-	res, err := lbAPI.CreateFrontend(createReq)
+	})
 	if err != nil {
 		return err
 	}
 
 	d.SetId(newRegionalId(region, res.ID))
 
+	err = resourceLbAclBetaUpdate(d, lbAPI, region, res.ID)
```

Comment:
You could simply do 
```
// We call update to handle creation of all ACL rules attached to this frontend
return resourceLbAclBetaUpdate(d, lbAPI, region, res.ID)
```

Reply:
So, if I recall it correctly (and looking at the code), with lb the update function will always force an update, so it's better to keep original approach (where the read is launched after the creation, and not the update). 

Also I believe there were some issues with the naming as well, caused by the fact that the state was not yet written at that point.